source/server/api/v1/_tests/projectRoutes.test.js

```diff
@@ -24,7 +24,7 @@ describe( 'couldDo routes', () => {
 
       it( 'should throw an error if no project with given id is found', () =>
         chai.request( server )
-          .post( '/project/198/could-dos' )
+          .get( '/project/198/could-dos' )
```

Comment:
Maybe change this to 'could-do' rather than 'could-dos', it looks like could 2 en español